compiler/grainformat/reformat.re

```diff
@@ -116,8 +116,7 @@ let get_last_item_in_list = comments =>
 let is_disable_formatting_comment = (comment: Parsetree.comment) => {
   switch (comment) {
   | Line(cmt) =>
-    if (String.trim(Comments.get_comment_source(comment))
-        == "// formatter-ignore") {
+    if (String.trim(cmt.cmt_source) == "// formatter-ignore") {
```

Comment:
But, why not `cmt.cmt_content == "formatter-ignore"`? There's not reason to check and trim the source.

Reply:
You are correct.  Fixed.